.gitignore

```diff
@@ -27,6 +27,7 @@ __pycache__
 *.swp
 .pytest_cache
 .python-version
+.venv*/
```

Comment:
You likely want that in your [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) to not have to do it for all your repos.